surveys/2023-annual-survey/questions.md

```diff
@@ -739,27 +744,43 @@ Rating:
 
 See [who](./design/who.md).
 
-The following are primarily for cohort analysis, secondarily for understanding the shape of the community.
+Your responses to the following questions will help our reviewers conduct cohort analyses and help us better understand the Rust community. Please answer according to your comfort level.  
 
-For methodological purposes, the bulk of the demographics should be at the end of the survey (unless acting as filter/flow questions above).
+~~For methodological purposes, the bulk of the demographics should be at the end of the survey (unless acting as filter/flow questions above).
 They're both easy to complete (beneficial at the end) and somewhat personal (but at this point folks are invested and we've built 'trust').
-Can also be problematic at start if we're asking all easy, personal questions and then get to the harder ones - easy to drop out.
+Can also be problematic at start if we're asking all easy, personal questions and then get to the harder ones - easy to drop out.~~
+
+### Do you hold a leadership role in the Rust Project or are you otherwise involved in strategic decision-making about Rust?
```

Comment:
There was some discussion about this here: https://github.com/rust-lang/surveys/issues/151, with some concerns that it might de-anonymize the answers.

To be honest, I don't personally consider this to be a useful question - there were almost 10k answers last year, so even if let's say ~100 members of Rust teams fill the survey (which I would find relatively unlikely), then it would still just be around 1% of all answers. But no problem from my side to keep it, of course.

Reply:
I'm also quite neutral. Fine for me to add this question as optional